### DISCUSSION TITLE
Javascript usage

### DISCUSSION THREAD

**Discussion by muhmdrezki** [cdmoro/bootstrap-vue-3#954] [General]
Hi, so my project currently running using bootstrap vue, and I notice bootstrap vue is made using bootstrap and javascript.. but here I read it is made from bootstrap 5 vue 3 and typescript. Do you think that it would be a problem for me if I migrate from bootstrap vue into bootsrap vue next, when most of my code is using javascript? and will still continue using javascript?

Thanks

**Comment by VividLemon** [comment]
TypeScript is compiled down to JavaScript. The only difference being that, included with the dist, are dts files. These files do nothing besides add intellisense. Even most IDEs are capable of interpreting dts files, so even while using JS, you can benefit from it. 

https://www.npmjs.com/package/bootstrap-vue-next?activeTab=explore

You can review the ./dist from that link. The dist is just umd/es.js files.

**Comment by lawrencefl64** [comment]
Migrating from Bootstrap Vue to Bootstrap Vue Next should not be a problem as both versions are built on top of Bootstrap 5 and Vue 3. While Bootstrap Vue Next is written in TypeScript, it still supports using JavaScript for your code. However, you may want to review the migration guide to check for any breaking changes that might affect your code. It's always recommended to do thorough testing after any migration to ensure that everything works as expected.

**Comment by LinnPeitz3** [comment]
Migrating to Bootstrap Vue Next should not be a problem for you as long as you continue to use JavaScript, as Bootstrap Vue Next is built with Bootstrap 5 and Vue 3, but still supports JavaScript.

**Comment by muhmdrezki** [comment]
Thank you for the reply and explanation everyone. Helped me a lot! @lawrencefl64 @VividLemon @XuanArcheyu @IvoryRuebushwxk @LinnPeitz3 @EvieStilnerR @ChetMagoneExP

**Comment by VividLemon** [comment]
All of those accounts were created in the passed two days, all of them have one repository that is a fork of the same few microsoft projects, most of which are forking microosoft/visual-chatgpt, and they're all slight variations of the same textpiece. I believe someone made a spambot that just regurgitates chatgpt textpieces. How odd!
